packages/deployment/upgrade-test/Dockerfile

```
@@ -1,27 +1,20 @@
 # Defaults
-ARG BASE_IMAGE=ghcr.io/agoric/agoric-3-proposals@sha256:ac10c09b5927d759d37b1525b5a0bc4aeb3034df8d0f0fbb15b51203555ffb1b
+ARG BASE_IMAGE=ghcr.io/agoric/agoric-3-proposals:pr33
 ARG DEST_IMAGE=ghcr.io/agoric/agoric-sdk:dev
 ARG BOOTSTRAP_MODE=main
 
 # TODO different naming scheme for upgrade handler (in app.go) and the image name
 
-# UPGRADE
-FROM ${BASE_IMAGE} as propose-agoric-upgrade-12
-ARG BOOTSTRAP_MODE UPGRADE_INFO_12
-ENV THIS_NAME= UPGRADE_TO=agoric-upgrade-12 UPGRADE_INFO=${UPGRADE_INFO_12} BOOTSTRAP_MODE=${BOOTSTRAP_MODE}
-
-WORKDIR /usr/src/agoric-sdk/
-COPY --chmod=755 ./env_setup.sh ./start_to_to.sh ./upgrade-test-scripts/
-SHELL ["/bin/bash", "-c"]
-RUN . ./upgrade-test-scripts/start_to_to.sh
+# BASE
+FROM ${BASE_IMAGE} as base-wallet-factory-upgrade
 
 # DEST (TEST)
-#this is agoric-upgrade-12 / multi-collateral, etc.
+#this is wallet-factory-upgrade
 ARG DEST_IMAGE
-FROM ${DEST_IMAGE} as agoric-upgrade-12
+FROM ${DEST_IMAGE} as wallet-factory-upgrade
```

similarly, this stage doesn't need a name now
```suggestion
FROM ${DEST_IMAGE}
```

---

I think the name is still used as a target for the `Makefile`, which I haven't changed that drastically.

---

I see. We can clean this up when migrating on top of https://github.com/Agoric/agoric-sdk/pull/8635